### PR TITLE
[MIG] Port hr_timesheet_task to 9.0

### PR DESCRIPTION
**Pull request by PierreFaniel**
Migrate the module hr_timesheet_task to Odoo v9.
Addition of tasks in timesheet report

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Hi, Pierre, thanks for the contribution. I have checked the CLA received.

File: hr_timesheet_task/models/hr_timesheet_report.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Niboo SPRL, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class HrTimesheetReport(models.Model):
+    _inherit = 'hr.timesheet.report'
+
+    task_id = fields.Many2one('project.task', 'Task', readonly=True)
```
Comment (Garamotte) [inline hr_timesheet_task/models/hr_timesheet_report.py]: Please explicitely name all arguments (`comodel_name` and `string`)

File: hr_timesheet_task/views/hr_timesheet_view.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```
Comment (Garamotte) [inline hr_timesheet_task/views/hr_timesheet_view.xml]: Replace `<openerp>` by `<odoo>` and remove the `<data>` tag.

File: hr_timesheet_task/__init__.py [hunk 3]
```diff
@@ -1,20 +1,4 @@
...
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2016 Niboo SPRL, Camptocamp SA
```
Comment (pedrobaeza) [inline hr_timesheet_task/__init__.py]: You should add a line per each author

File: hr_timesheet_task/__openerp__.py [hunk 4]
```diff
@@ -1,43 +1,29 @@
...
-# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
+{
+    'name': 'HR - Task In Timesheets',
+    'category': 'HR',
+    'summary': 'Allow the user to select task in a timesheet',
+    'website': 'https://odoo-community.org/',
+    'license': 'AGPL-3',
+    'version': '9.0.1.0.0',
+    'description': '''
+    The user is now able to select a task in a timesheet
```
Comment (pedrobaeza) [inline hr_timesheet_task/__openerp__.py]: Extract this to a README.rst and fill the complete template like https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: hr_timesheet_task/__openerp__.py (hunk 4 above)
Comment (rafaelbn) [reply hr_timesheet_task/__openerp__.py]: Please @PierreFaniel take care of this issue when writing README https://github.com/OCA/hr-timesheet/issues/54 and thanks for this contribution!

Comment (pedrobaeza) [reply hr_timesheet_task/__openerp__.py]: timesheet_task module can be removed now in v9.

File: hr_timesheet_task/static/description/index.html [hunk 5]
```diff
@@ -15,12 +15,12 @@ <h2 class="oe_slogan">Credits</h2>
         <div class="oe_span12">
```
Comment (pedrobaeza) [inline hr_timesheet_task/static/description/index.html]: Now that you have put README.rst, you can remove this file.

**Comment by rafaelbn** [comment]
Runbot fails, I cannot test it: skipped state in https://runbot.odoo-community.org/runbot/repo/github-com-oca-hr-timesheet-117

**Comment by PierreFaniel** [comment]
Hello,

I would like to know why runbot skips all the v9 builds for this repo.

Can anybody help with this ?

**Comment by pedrobaeza** [comment]
It should be because .travis.yml is not according last template: https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml

**Comment by PierreFaniel** [comment]
This pull request seems to not want to work for me 😄 

Travis fails and in the logs I see
`ParseError: "13 Permission denied" while parsing /home/travis/odoo-9.0/openerp/addons/base/base_data.xml:26`
but I don't understand why. Is this something I caused? Is the .travis.yml wrongly configured?

**Comment by pedrobaeza** [comment]
I think it's due to https://github.com/odoo/odoo/pull/15248.

**Review by feketemihai — APPROVED** [review]
Not tested

**Review by rafaelbn — APPROVED** [review]
Thanks @PierreFaniel :+1: 

There are still some minor comments that you could quicky attend.

**Pull request closed without merging**